### DISCUSSION TITLE
aarch32 support

### DISCUSSION THREAD

**Discussion by Zuzu-Typ** [pypa/cibuildwheel#961] [Ideas]
I'd like to suggest adding support for **aarch32** (ARMv7) builds to cibuildwheel. Although this architecture has been superseded by aarch64 (ARMv8), a lot of devices still use aarch32. First and foremost the **Raspberry Pi** line of products (or more specifically, their default OS).

If I understand the architecture correctly, ARMv8 chips are also backwards-compatible with aarch32, meaning the latter should run on those devices too.

What are your thoughts on this?
(I'm sorry if I have overlooked this being suggested before. I only did a brief search)

**Comment by joerick** [comment]
If you're looking for wheels for Raspberry Pi, I recommend you check out the [piwheels](https://www.piwheels.org/) project. They're set up for this exact use case.

**Reply by joerick** [reply]
Otherwise, it would require somewhere to run the CI, a new [manylinux](https://github.com/pypa/manylinux) image, and support here. The question would be if it's worth the community effort to support another platform, given that the piwheels project seems to be doing such a great job!
